coalahtml/_coalahtml/index.html

```diff
@@ -11,15 +11,15 @@
     <link rel="stylesheet"
           href="https://fonts.googleapis.com/css?family=Roboto:100,200,300,400,500,600,800,900">
     <link rel="stylesheet"
-          href="bower_components/bootstrap/dist/css/bootstrap.css">
-    <link rel="stylesheet"
           href="bower_components/google-code-prettify/src/prettify.css">
-    <!-- Bootstrap Material Design -->
     <link rel="stylesheet"
-          href="bower_components/bootstrap-material-design/dist/css/bootstrap-material-design.css">
+          href="bower_components/materialize/dist/css/materialize.min.css">
```

Comment:
Any reasons for migrating to materializecss? Please mention that in the commit message.